### DISCUSSION TITLE
No Response to GitHub Student Update

### DISCUSSION THREAD

**Discussion by nasokin** [community/community#130796] [GitHub Education]
### Hi everyone,

Dear GitHub Team,
 
I have submitted my application of updating my GitHub Student more than 2 weeks ago but still got no response.

**Comment by emansarahafi** [comment]
Hello, sometimes it takes longer for the update to be processed, some users receive an update within 3 days but it can take a while during long periods. I suggest you reach out to someone from the GitHub Education staff who could fix it for you by opening a ticket [here](https://support.github.com/contact/account), selecting Other, and explaining your situation.
